DataFormats/Detectors/TOF/include/DataFormatsTOF/Diagnostic.h

```diff
@@ -49,7 +49,8 @@ class Diagnostic
   void fill(const Diagnostic& diag);                       // for calibration
   void fill(const gsl::span<const o2::tof::Diagnostic>){}; // for calibration
   void merge(const Diagnostic* prev);
-  void getNoisyMap(Bool_t* output); // set true in output channel array
+  void getNoisyMap(Bool_t* output, int noisyThr = 1); // set true in output channel array
+  void getNoisyLevelMap(Char_t* output);              // set true in output channel array
```

Comment:
Aren't these const methods?

Reply:
Yes, they are. Moved to const (including also other non static getters).